alf/algorithms/rl_algorithm.py

```diff
@@ -87,12 +90,17 @@ def __init__(self,
             action_spec (nested BoundedTensorSpec): representing the actions.
             train_state_spec (nested TensorSpec): for the network state of
                 ``train_step()``.
+            reward_spec (TensorSpec): a rank-1 or rank-0 tensor spec representing
+                the reward(s).
```

Comment:
need to assert reward_spec is rank-1 or rank-0

Reply:
added